hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/operations/AuthenticationFailureOp.java

```diff
@@ -21,7 +21,10 @@
 import com.hazelcast.logging.ILogger;
 import com.hazelcast.spi.impl.NodeEngineImpl;
 
-public class AuthenticationFailureOperation extends AbstractClusterOperation {
+public class AuthenticationFailureOp extends AbstractClusterOperation {
+
+    public AuthenticationFailureOp() {
```

Comment:
Constructors cannot be overridden. `super()` will be implicitly called regardless of defining empty c'tor. Most of the operations already have empty c'tors.